pkg/chartmuseum/router/middleware.go

```diff
@@ -76,7 +76,7 @@ func setupContext(c *gin.Context) {
 	c.Set("requestcount", reqCount)
 	reqID := c.Request.Header.Get("X-Request-Id")
 	if reqID == "" {
-		reqID = uuid.NewV4().String()
+		reqID = uuid.Must(uuid.NewV4()).String()
```

Comment:
Can we remove the `Must` here? should be unnecessary, only used for things like
```
var packageUUID = uuid.Must(uuid.FromString("123e4567-e89b-12d3-a456-426655440000"));
```
according to https://godoc.org/github.com/gofrs/uuid#Must
`NewV4` should be good

Reply:
I think we might need it, otherwise we would need to figure out how to bubble up the error that is return. `Must` swallows that error by panicking should the err be non-nil.